staff.php

```diff
@@ -292,6 +300,24 @@ function addStaff($staff, &$error, &$message)
 
 		// is a day provided?
 		if (trim($staff_array[2])) {
+			// provide defaults
+			if (!$staff_array[3]) {
+				// default earliest time
+				$staff_array[3] = '12:00 am';
+			}
+			if (!$staff_array[4]) {
+				// default latest time
+				$staff_array[4] = '11:59 pm';
+			}
+			if (!$staff_array[5]) {
+				// default language id
```

Comment:
This has been updated in the latest commit - somehow we are viewing outdated code.